js/app.js

```diff
@@ -1,9 +1,76 @@
-// wait for DOM to load before running JS
 $(document).ready( function() {
 
-  // check to make sure JS is loaded
-  console.log('JS is loaded!');
+//click event
+	$('#search-form').on('submit', function(event) {
+		event.preventDefault();
+//search value
+		let search = $('#search-field').val();
+//ajax call to api
+		$.ajax({
+			type:"GET",
+			url: 'https://api.spotify.com/v1/search',
+			headers: {
+				'Authorization': 'Bearer ' + "BQA5E9jKRcEDF_BSH0Do4zF_0rE4lRqQWQd067bgoLn_5KXl6AOuXQDg6DztkFPQ6MKozPAx6H7WkiIAi-V_7Q"
+			},							
+			data: {
 
-  // your code here
+				q: search,
+				type:"track"
+
+			},
+		})
+//response data
+		.then(function(response) {
+
+//loop thru data
+			response.tracks.items.forEach((el, i)=>{
+				console.log(el.preview_url);
+				console.log(el.name);
+				console.log(el.album.images[0].url);
+				console.log(el.artists[0].name);
+
+//append elements w/ data			
+			$('#results').append(`
+					<div class="container full">
+					<div class="col-sm-4 box">
+					<h2>${el.artists[0].name}</h2>
+					<h5>${el.name}</h5>
+					<img class="artistImg" src="${el.album.images[0].url}">
+
+					<audio id="audio${i}" src="${el.preview_url}"></audio>
+					<input type="button" value="PLAY" onclick="play(${i})">
+					<input type="button" value="PAUSE" onclick="pauseAudio(${i})">
+					</div>
+					</div>
+					`);			
+			});
```

Comment:
Hey Bryan,

Great job as always. Code looks good, just one note:

I think it's a good idea to always make sure your code is properly indented - this includes html in a template literal. 

That's weird about the audio thing, but I'm glad you found a way to make it work.

Matt